scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -204,16 +230,20 @@ task CreateImportTsvs {
         -SNM ~{sample_map} \
         --ref-version 38
 
+      if [ ~{has_service_account_file} = 'true' ]; then
```

Comment:
If you moved this up above the GATK invocation then I believe GATK can read directly from the source bucket, without localizing, and using the right credentials